src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/commands.py

```diff
@@ -146,6 +146,7 @@ def load_command_table(self, _):
         g.custom_command('list', 'list_images')
         g.show_command('show', 'get')
         g.command('delete', 'delete')
+        g.generic_update_command('update')
```

Comment:
Why are you not exposing any convenience arguments?

Reply:
None in particular came to mind, I will add tags as you've requested below. I didn't bother adding it, as vm update does not add `--tags`